masterlist.yaml

```diff
@@ -5741,14 +5396,7 @@ plugins:
       - name: 'Enhanced Lights and FX'
         condition: 'active("EnhancedLightsandFX.esp") and not active("ELFX - NoPlayerHomes.esp")'
     msg:
-      - <<: *hasWildEdits
-        subs:
-          - |+
-
-              - Open SSEEdit and load **Distinct Interiors - Player Homes.esp**.
```

Comment:
Same as above, removing this loses information.

Reply:
https://github.com/loot/skyrimse/commit/2acab29df606cad6755c3d5eb565de684ee322d5